how2meet/ui/pages/events.py

```diff
@@ -0,0 +1,139 @@
+async def events(user: str = None):
+    """List page for all events"""
+    with frame("Events"):
+        # Get the list of events from the API
+        # NOTE: This is a blocking call, so we use an async client
+        async with httpx.AsyncClient() as client:
+            events = await client.get(f"http://localhost:8000/api/events/", timeout=10)
+        try:
+            events = events.json()
+        except:
+            events = []
+
+        async def delete_event(event_id: str):
+            async with httpx.AsyncClient() as client:
+                response = await client.delete(f"http://localhost:8000/api/events/{event_id}", timeout=10)
+                response.raise_for_status()
+            ui.open("/events")
+
+        # Display the list of events
+        for event in events:
+            with ui.card().classes("w-full"):
+                with ui.row().classes("w-full justify-between"):
+                    with ui.column().classes("flex-grow"):
+                        ui.label(f"Event ID: {event['id']}")
+                        ui.label(f"Event Name: {event['name']}")
+                    ui.button("", icon="info", on_click=lambda event_id=event['id']: ui.open(f"/events/{event_id}")).classes("w-6 h-6")
+                    ui.button("", icon="delete", color="red", on_click=lambda event_id=event['id']: delete_event(event_id)).classes("w-6 h-6")
+
+        # Add navigation buttons
+        with ui.row().classes("w-full justify-center"):
+            ui.button("Back", on_click=lambda: ui.open("/"))
+            ui.button("New Event", on_click=lambda: ui.open(f"/events/new/{uuid.uuid4()}"))
+
+
+@router.page("/{event_id}")
+async def event_home(event_id: str):
+    """Detail page for a specific event"""
+    # Get the event from the API
+    # NOTE: This is a blocking call, so we use an async client
+
+    with frame("Event Home"):
+        async with httpx.AsyncClient() as client:
+            event = await client.get(f"http://localhost:8000/api/events/{event_id}", timeout=10)
+        try:
+            event = event.json()
+        except:
+            event = {}
+
+        # Display the event details
+        with ui.column().classes("w-full justify-between"):
+            for key, value in event.items():
+                ui.label(f"{key}: {value}")
+        ui.button("Back", on_click=lambda: ui.open("/events"))
+
+@router.page("/new/{event_id}")
+async def new_event(event_id: str):
+    """New event creation page
+
+    Args:
+        event_id (str): The unique ID of the event to be created.
+
+    Returns:
+        None
+    """
+
+
+    async def add_guest():
+        invite_uuid = str(uuid.uuid4())
+        # TODO add method to remove guest
+        with ui.card().classes("flex-grow position:relative") as card:
+            async def delete(invite_uuid=invite_uuid):
+                card.delete()
+            with ui.row().classes("flex-grow justify-between items-center"):
+                name_input = ui.input("Name")
```

Comment:
I'm unclear on how this differs from `.bind_value()` 

I prefer what you have here since it is more intuitive, but I'm wary that there's some small but important differences for how they work. Such as when the data updates, whether is reactive after focus has switched or if it requires a "post"

Reply:
Consider this unfinished work. This ticket didn't address invites. I just wanted something somewhat working there.